deploy/009-configure-vaults.ts

```diff
@@ -0,0 +1,67 @@
+import deployedContracts from "@venusprotocol/governance-contracts/deployments/deployments.json";
```

Comment:
I think this json file is not available in the package `@venusprotocol/governance-contracts`. Moreover, I would update the dependency in the package.json, to use the version `1.4.0-dev.2`, that includes the ACM deployed to Ethereum

Reply:
I chose to resolve this by adding deployments to the hardhat config. The governance-contracts repo version has also been bumped